app/Models/Enums/Gender.php

```diff
@@ -6,9 +6,9 @@ enum Gender: string
 {
   use EnumToArray;
 
-    case MAN = 'Man';
-    case VROUW = 'Vrouw';
-    case ANDERS = 'Anders';
-    case NIETVANTOEPASSING = 'N.v.t.';
+    case Man = 'Man';
+    case Vrouw = 'Vrouw';
+    case Anders = 'Anders';
+    case NietVanToepassing = 'N.v.t.';
```

Comment:
Enums moeten volledig uppercase zijn afaik.

```suggestion
    case MAN = 'Man';
    case VROUW = 'Vrouw';
    case ANDERS = 'Anders';
    case NIETVANTOEPASSING = 'N.v.t.';
```

Reply:
volgens Cas gaat het hierbij met cammelcase